daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -7,6 +7,11 @@ package btrfs // import "github.com/docker/docker/daemon/graphdriver/btrfs"
 #include <stdlib.h>
 #include <stdio.h>
 #include <dirent.h>
+#include <linux/version.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4,12,0)
+#error "Headers from kernel >= 4.12 are required on compilation time (not on run time)"
+#error "Hint: set 'DOCKER_BUILDTAGS=exclude_graphdriver_btrfs' to exclude btrfs"
+#endif
```

Comment:
```suggestion

#include <linux/version.h>
#if LINUX_VERSION_CODE < KERNEL_VERSION(4,12,0)
    #error "Headers from kernel >= 4.12 are required to build with Btrfs support."
    #error "HINT: Set 'DOCKER_BUILDTAGS=exclude_graphdriver_btrfs' build without Btrfs."
#endif
```

Reply:
done

Reply:
Looks like I managed to drop a word -- `to build without Btrfs.`

Also, could we add the line break between the libc and kernel version header?

Reply:
done